### PR TITLE
Update AboutUs

### LINKED ISSUE #25: Update AboutUs page
Standardised format:

Links: only a github link to your profile
Roles: Developer

### PR DESCRIPTION
**Pull request by VaishakAnand**

### CODE REVIEW

**Review by csiongn — APPROVED** [review]

File: docs/AboutUs.md [hunk 1]
```diff
@@ -19,15 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-### Jane Doe
+### Vaishak Anand
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/vaishakanand.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](http://github.com/VaishakAnand)]
 [[portfolio](team/johndoe.md)]
```
Comment (ong-yinggao98) [inline docs/AboutUs.md]: Might want to remove this line since everyone else also did so

**Review by ong-yinggao98 — COMMENTED** [review]
See comments pls

File: docs/AboutUs.md (hunk 1 above)
Comment (VaishakAnand) [reply docs/AboutUs.md]: okay, removed

**Review by AlexCQY — APPROVED** [review]
LGTM

**Review by hogantan — APPROVED** [review]
LGTM

**Review by ong-yinggao98 — APPROVED** [review]
LTGM

**Pull request merged**